scripts/validate_deployment.py

```
@@ -0,0 +1,335 @@
+            print("❌ Verify disabled: FAIL")
+            return False
+        
+        # Test enable
+        result = self._make_request("POST", "network-profiling/enable")
+        if not result["success"] or result["status_code"] != 200:
+            print(f"❌ Enable profiling: FAIL - {result.get('error', 'Unknown error')}")
+            return False
+        
+        # Verify enabled
+        result = self._make_request("GET", "network-profiling/status")
+        if not result["success"] or not result["data"].get("enabled") == True:
```

Ruff (0.12.2)

Use `result["data"].get("enabled") != True` instead of `not result["data"].get("enabled") == True`

Replace with `!=` operator

(SIM201)